### PR TITLE
Fix fork tests

### PR DESCRIPTION
**Pull request by shahthepro**
If you made a contract change, make sure to complete the checklist below before merging it in master.

Refer to our [documentation](https://github.com/OriginProtocol/security) for more details about contract security best practices.

Contract change checklist:
  - [ ] Code reviewed by 2 reviewers. 
  - [ ] Copy & paste code review [security checklist](https://github.com/OriginProtocol/security/blob/master/templates/Contract-Code-Review-Example.md) below this checklist.
  - [ ] Unit tests pass
  - [ ] Slither tests pass with no warning
  - [ ] Echidna tests pass if PR includes changes to OUSD contract (not automated, run manually on local)

### CODE REVIEW

File: contracts/test/strategies/ousd-metapool-ousd-tilted-pool.fork-test.js [hunk 1]
```diff
@@ -22,24 +22,12 @@ describe("ForkTest: Convex 3pool/OUSD Meta Strategy - Titled to OUSD", function
       await mintTest(fixture, josh, usdt, "100000");
     });
 
-    describe("Mint", function () {
-      it("Should stake USDT in Curve gauge via metapool", async function () {
```
Comment (sparrowDom) [inline contracts/test/strategies/ousd-metapool-ousd-tilted-pool.fork-test.js:26]: Are we leaving the USDT mint test out?

File: contracts/test/vault/oeth-vault.fork-test.js [hunk 2]
```diff
@@ -18,143 +18,140 @@ describe("ForkTest: OETH Vault", function () {
 
   let fixture;
 
-  describe("OETH Vault", () => {
```
Comment (sparrowDom) [inline contracts/test/vault/oeth-vault.fork-test.js:21]: any reason we are removing the top level describe?

**Review by sparrowDom — COMMENTED** [review]
Couple of questions otherwise LGTM

File: contracts/test/strategies/ousd-metapool-ousd-tilted-pool.fork-test.js (hunk 1 above)
Comment (shahthepro) [reply contracts/test/strategies/ousd-metapool-ousd-tilted-pool.fork-test.js:26]: There's a duplicate test just below this line

File: contracts/test/vault/oeth-vault.fork-test.js (hunk 2 above)
Comment (shahthepro) [reply contracts/test/vault/oeth-vault.fork-test.js:21]: On the CI, we split tests upto level-1. Here everything nested to level-2 without any specific reason. So making them level-1 will make sharing of test cases across the runners a bit more balanced

File: contracts/test/strategies/ousd-metapool-ousd-tilted-pool.fork-test.js (hunk 1 above)
Comment (sparrowDom) [reply contracts/test/strategies/ousd-metapool-ousd-tilted-pool.fork-test.js:26]: got it thanks

**Pull request merged**